cfgov/unprocessed/css/molecules/featured-menu-content.less

```diff
@@ -0,0 +1,55 @@
+/* topdoc
+  name: Featured Menu Content
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+          <aside class="m-featured-menu-content">
+              <a href="/the-bureau/">
+                  <img src="/static/img/fmc-about-us-540x300.jpg"
+                       alt="About Us video image"
+                       height="150"
+                       width="270">
+                  <p class="h4 u-mt15">
+                    The CFPB: Working for you
+                  </p>
+              </a>
+              <p>
+                  Short description.
+              </p>
+          </aside>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .m-featured-menu-content
+            a
+              img
+              p.h4.u-mt15
+            p
+  tags:
+    - cfgov-molecules
+*/
+
+.m-featured-menu-content {
+    .grid_column( 4 );
+
+    padding: 0 @grid_gutter-width / 2;
```

Comment:
Yep, the font-size hasn't been changed here so you're fine to use the `@base-font-size-px` var.